contracts/stabilizer/IPegStabilityModule.sol

```diff
@@ -88,4 +89,7 @@ interface IPegStabilityModule {
 
     /// @notice the PCV deposit target to send surplus reserves
     function target() external view returns (IPCVDeposit);
+
+    /// @notice the balance of the reserve asset
+    function tokenBalance() external view returns (uint256);
```

Comment:
Why add another method? @ditchfieldcaleb did this too. Is the IPCVDeposit interface not intuitive? Too late to change name of balance() function